dex.dev.yml

```diff
@@ -11,10 +11,12 @@ web:
 staticClients:
 - id: signals
   redirectURIs:
-    - 'http://localhost:3001/manage/incidents'
-    - 'http://127.0.0.1:3001/manage/incidents'
-  name: 'Signals'
-  secret: ZXhhbXBsZS1hcHAtc2VjcmV0
+    - http://localhost:3001/manage/incidents
+    - http://127.0.0.1:3001/manage/incidents
+    - http://localhost:8001/signals/oidc/callback/
+    - http://127.0.0.1:8001/signals/oidc/callback/
```

Comment:
Can you also add port `8000`, when running from docker-compose it spins up the API with a port mapping on `8000:8000`

Reply:
@vanbuiten the button is hidden when `OIDC_RP_CLIENT_ID` is not configured, right?

Reply:
@bartjkdp Ah yes I see. You are correct, when I do not configure the `OIDC_RP_CLIENT_ID` the button is gone. And when configured the button is there.

Reply:
Added port `8000` to dex.dev.yml